conformance_pack/wafv2.sp

```diff
@@ -10,7 +10,6 @@ control "wafv2_web_acl_logging_enabled" {
   query       = query.wafv2_web_acl_logging_enabled
 
   tags = merge(local.conformance_pack_wafv2_common_tags, {
-    cis_controls_v8_ig1                    = "true"
```

Comment:
We should keep this tag. This control is being used in `cis_controls_v8_ig1_8_2`